### ISSUE TITLE
global: consistent date format 

### ISSUE THREAD

**Issue by topless** [inveniosoftware/invenio-circulation#106]
* ElasticSearch `date` field should probably be changed
* all loan metadata dates should be stored in their ISO format
* ~~`_created`, `_updated` should be removed from mappings~~

**Comment by BadrAly** [comment]
1. ElasticSearch date field should probably be changed
Can you give us an additional information on what you want to do here.

2. _created, _updated should be removed from mappings
Normalizing the loan dates seems to be a good idea.

3. all loan metadata dates should be stored in their ISO format
We just do not understand the reason behind removing the _created, _updated from the loan mapping.

In any case, we will be definitely interested to see you pull request 😀

**Comment by ntarocco** [comment]
Currently `_created` and `_updated` are set like [this](https://github.com/inveniosoftware/invenio-indexer/blob/master/invenio_indexer/api.py#L299):
```
        data['_created'] = pytz.utc.localize(record.created).isoformat() \
            if record.created else None
        data['_updated'] = pytz.utc.localize(record.updated).isoformat() \
            if record.updated else None
```

**Comment by ntarocco** [comment]
solved in #119
